Italian/smartphone/facebook/primo/primo.md

```diff
@@ -0,0 +1,7 @@
+Ciao, scusa se ti disturbo, ma ti ricordi di me?
+
+Caspita, ma tutti a me mi dovete capitare? Soltanto perché il mio nome comincia per A ed è il primo della lista vi sentite 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.